pkg/minikube/node/cache.go

```diff
@@ -193,6 +193,8 @@ func waitDownloadKicBaseImage(g *errgroup.Group) {
 			}
 			if errors.Is(err, image.ErrGithubNeedsLogin) || errors.Is(err, image.ErrNeedsLogin) {
 				exit.Message(reason.Usage, `Please either authenticate to the registry or use --base-image flag to use a different registry.`)
+			} else if err.Error() == "failed to download kic base image or any fallback image" {
```

Comment:
I am not convinced what this PR is trying to actually do ? do u mind sharing before and after this PR with reproduce-able steps?

Reply:
@medyagh I'm having a hard time reproing this issue given that occurs because of connectivity issues. any tips? "failed to download kic base image or any fallback image" string is what is returned when begin Download function can't pull the appropriate images. This provides a mechanism to match with that and exit out.

Feel free to close the issue if you don't think this is a problem anymore.

Reply:
u can make the func to Return Error to mock it